experimental/dds/tree/src/EditLog.ts

```diff
@@ -425,27 +469,24 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 		info?: EditSequencingInfo,
 		minSequenceNumber: number = 0
 	): void {
-		assert(
-			minSequenceNumber >= this.minSequenceNumber,
-			'Sequenced edits should carry a monotonically increasing min number'
-		);
-		// The new minSequenceNumber indicates that no future edit will require information from edits with a smaller or equal seq number
-		// for its resolution.
-		this._minSequenceNumber = minSequenceNumber;
-		// TODO:#57176: Increment maximumEvictableIndex to reflect the fact we can now evict edits with a sequenceNumber lower or equal to
-		// it. Note that this will change the meaning of our 'InSession' APIs so we should make sure to rename them at the same time.
-		// The code might look like this:
-		// while (this.maximumEvictableIndex + 1 < this.indexOfFirstEditInSession) {
-		// 	const nextEdit = this.getEditInSessionAtIndex(this.maximumEvictableIndex + 1);
-		// 	const nextEditInfo = this.getOrderedEditId(nextEdit.id) as SequencedOrderedEditId;
-		// 	if (
-		// 		nextEditInfo.sequenceInfo !== undefined &&
-		// 		nextEditInfo.sequenceInfo.sequenceNumber > minSequenceNumber
-		// 	) {
-		// 		break;
-		// 	}
-		// 	++this.maximumEvictableIndex;
-		// }
+		// The index of the sequenced edit to add
+		const index = this._earliestAvailableEditIndex + this.numberOfSequencedEdits;
+
+		if (info !== undefined) {
+			assert(
+				minSequenceNumber >= this._minSequenceNumber,
+				'Sequenced edits should carry a monotonically increasing min number'
+			);
+			this._minSequenceNumber = minSequenceNumber;
+
+			// Calculate the index of the minSequenceNumber by comparing it to the index of this sequenced edit
+			const sequenceNumberDifference = info.sequenceNumber - minSequenceNumber;
```

Comment:
nit: Could you note here that `sequenceNumberDifference` === "the size of the current collab window" either by rename or comment? It took my smol brain a good couple of minutes to figure out what this was doing.

Reply:
Resolving cause this is no longer needed.